app/src/main/java/org/mozilla/tv/firefox/IntentValidator.kt

```diff
@@ -30,6 +31,8 @@ data class ValidatedIntentData(val url: String, val source: Session.Source)
 object IntentValidator {
     @VisibleForTesting const val DIAL_PARAMS_KEY = "com.amazon.extra.DIAL_PARAM"
     private const val ACTIVE_EXPERIMENTS_KEY = "activeExperiments"
+    private const val IS_QA_BUILD_KEY = "isQA"
```

Comment:
nit: I prefer to declare private constants at the top level, which reduces the size of the object's block, making it clearer what is important in this class.

I think liuche was going to move the existing constant to the top level in a follow-up.